pyproject.toml

```diff
@@ -6,20 +6,29 @@ build-backend = "setuptools.build_meta"
 line-length = 80
 
 [tool.coverage.run]
-source = ["zoneinfo"]
 omit = ["tests/typing_example.py"]
 
 [tool.coverage.paths]
 source = ["src", ".tox/*/site-packages"]
 
 [tool.coverage.report]
 show_missing = true
-skip_covered = true
+skip_covered = false
 
 [tool.isort]
 atomic=true
 force_grid_wrap=0
 include_trailing_comma=true
+known_first_party = ["backports.zoneinfo"]
+known_third_party=[
+    "click",
+    "dateutil",
+    "hypothesis",
+    "pint",
+    "pytest",
+    "pytz",
+    "requests",
+]
```

Comment:
Maybe just configure `isort` to [assume that anything unknown is third-party](https://github.com/Zac-HD/hypothesis-jsonschema/blob/0ca95cf61ab52f7a7edb72721f82ac362bc8da5d/tox.ini#L65-L67)?

Reply:
If I do this, `from backports import zoneinfo` gets sorted as 3rd party, even though `import backports.zoneinfo` gets sorted as first-party, strangely enough.